pkg/sdk/privileges.go

```diff
@@ -5,7 +5,7 @@ type Privilege string
 const (
 	PrivilegeUsage          Privilege = "USAGE"
 	PrivilegeSelect         Privilege = "SELECT"
-	PrivilegeReferenceUsage Privilege = "REFERENCE USAGE"
+	PrivilegeReferenceUsage Privilege = "REFERENCE_USAGE"
```

Comment:
Can't find REFERENCE_USAGE in https://docs.snowflake.com/en/sql-reference/sql/grant-privilege